### ISSUE TITLE
Use GitFlow-style deployment to cloud.gov

### ISSUE THREAD

**Issue by LindsayYoung** [GSA-TTS/FAC#581]
After the team agrees on #580 We will want to align our deployment to match our strategy.  This ticket will be updated accordingly.

### Background
We are switching to a GitFlow based model.

### Acceptance Criteria (We'll know we're done when...)
- [ ] The app is running in the  FAC org's dev space with automated deployment
- [ ] The app is running in the  FAC org's staging space with automated deployment
- [ ] The app is running in the  FAC org's prod space with automated deployment
- [ ] GItHub permissions and settings are reviewed to match the criteria laid out in #580

### Tasks
Set up dev space
- [ ] Secrets transferred from the existing org to FAC cloud.gov org
- [ ] Create a dev space in cloud.gov
- [ ] Create a service key in the dev space to deploy with GitHub Actions
- [ ] Create a `develop` branch from the current version of `main` in GitHub
- [ ] Create a deployment workflow with GitHub Actions so that once a change is merged into the `develop` branch and the tests pass the code is pushed and built in the development  space
- [ ] Change the settings in GitHub so by default, PRs will target the `develop` branch

Set up staging space
- [ ] Secrets transferred from the existing org to FAC cloud.gov org
- [ ] Create a staging space in cloud.gov
- [ ] Create a service key in the staging space to deploy with GitHub Actions
- [ ] Create a `staging` branch from the current version of `main` in GitHub
- [ ] Create a deployment workflow with GitHub Actions so that once a change is merged into  the `staging` branch and the tests pass, the code is pushed and built in the `staging` space

Set up Prod space
- [ ] Secrets transferred from the existing org to FAC cloud.gov org
- [ ] Create a space in cloud.gov
- [ ] Create a service key in the __ space to deploy with GitHub Actions
- [ ] Create a deployment workflow with GitHub Actions so that once a change is merged into `main` branch and the tests pass, the code is pushed and built in the `production` space
